pkg/odo/cli/component/create.go

```diff
@@ -142,15 +143,16 @@ A full list of component types that can be deployed is available using: 'odo cat
 		exists, err := catalog.Exists(client, componentType)
 		odoutil.CheckError(err, "")
 		if !exists {
-			fmt.Printf("Invalid component type: %v\nRun 'odo catalog list components' to see a list of supported component types\n", componentType)
+			log.Errorf("Invalid component type: %v", componentType)
+			log.Info("Run 'odo catalog list components' for a list of supported component types")
 			os.Exit(1)
 		}
 
 		// Check to see if that particular version exists
 		versionExists, err := catalog.VersionExists(client, componentType, componentVersion)
 		odoutil.CheckError(err, "")
 		if !versionExists {
-			fmt.Printf("Invalid component version: %v\nRun 'odo catalog list components' to see a list of supported component type versions\n", componentVersion)
+			log.Errorf("Invalid component version: %v\nRun 'odo catalog list components' to see a list of supported component type versions", componentVersion)
```

Comment:
Why not break the statement into two as you did right above?

Reply:
You're right, we should!